vcpkg/reference/installation-tree-layout.md

```diff
@@ -0,0 +1,98 @@
+`$VCPKG_ROOT` with your vcpkg installation path). In manifest mode, each manifest file has a
+corresponding `vcpkg_installed` directory. Regardless of the operation mode, the layout of the
+installation directory remains the same.
+
+The installation directory is created the first time a package is installed, if you don't see a
+installation directory try installing some packages first.
+
+The root level of the installation directory contains:
+
+* A `vcpkg` directory, the vcpkg database, keeps track of installed packages and files
+* A directory per each [triplet](../users/triplets.md) that has installed packages, contains the
+  packages build output
+
+## `vcpkg` directory
+
+The vcpkg directory contains two subfolders (`info` and `updates`) and a file named `status`:
+
+### `info` directory
+
+Contains the list of files installed by each port, the naming convention for these files is: 
+`<port name>_<version>_<triplet>.list`. The primary purpose of these files is to avoid conflicts
+between ports attempting to install files with the same name. The [`vcpkg
+owns`](../commands/owns.md) can be used to query the information in these files.
+
+### `status` file
+
+The vcpkg database, this file contains an entry for every package installed for every triplet. This
+file is automatically updated by vcpkg and should not be modified manually.
+
+## Triplet directories
+
+The output of each package installation is contained in a triplet-specific directory, for example,
+packages installed for the `x64-windows` triplet are located in `installed/x64-windows` directory.
+
+The layout for the subdirectories inside each triplet directory is the same:
+
+| Subdirectory            | File type                                         | 
+| ----------------------- | ------------------------------------------------- |
+| `bin`                   | Release `.dll` and `.pdb` files                   |
+| `debug/bin`             | Debug `.dll` and `.pdb` files                     |
+| `debug/lib`             | Debug `.lib`, `.so`, and `.a` files               |
+| `debug/lib/manual-link` | Manually-linkable debug `.dll` and `.pdb` files   |
+| `debug/lib/pkgconfig`   | pkgconfig files (`.pc`)                           |
+| `include`               | Header-files (`.h`, `.hpp`, `.hxx`)               |
+| `lib`                   | Release `.lib`, `.so`, and `.a` files             |
+| `lib/manual-link`       | Manually-linkable release `.dll` and `.pdb` files |
+| `lib/pkgconfig`         | Pkgconfig files (`.pc`)                           |
+| `share/<port>`          | Miscellaneous files.                              |
+| `share/pkconfig`        | pkgconfig files (`.pc`)                           |
+| `tools/<port>`          | Executable tools                                  |
+
+
+### `bin` and `debug/bin` directories
+
+On Windows, these directories contain DLL and PDB files for release and debug configuration
+respectively.
```

Comment:
I would add a note that executables should be moved to `tools/<port>/` and not be placed in the `bin/` directory.